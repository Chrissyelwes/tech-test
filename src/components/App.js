import React from "react";
import "../styles/app.css";
import Search from "../components/Search";

const App = () => {
  return (
    <div className="app">
        <img className="logo"
        src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg" alt="NASA logo"
         />
      <Search />
    </div>
  );
}

export default App;