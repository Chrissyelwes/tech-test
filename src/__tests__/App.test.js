import React from "react";
import { render, screen } from '@testing-library/react';
import App from '../components/App';

describe("App", () => {
    const { asFragment } = render(<App />)

    test("renders App component correctly", () => {
        expect(asFragment()).toMatchSnapshot();
    });
});