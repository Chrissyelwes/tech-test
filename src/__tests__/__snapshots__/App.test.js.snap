// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders App component correctly 1`] = `
<DocumentFragment>
  <div
    class="app"
  >
    <img
      alt="NASA logo"
      class="logo"
      src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
    />
    <div
      class="search"
    >
      <form
        class="search-form"
      >
        <input
          class="search-input"
          type="text"
        />
        <button
          class="search-btn"
          type="submit"
        >
          Search
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
